mscore/shortcut.cpp

```diff
@@ -2771,6 +2771,30 @@ Shortcut Shortcut::_sc[] = {
          Qt::WindowShortcut,
          ShortcutFlags::A_CHECKABLE
          },
+#ifdef AVSOMR
+    {
+       MsWidget::MAIN_WINDOW,
+       STATE_NORMAL,
+       "avsomr-hide-all",
+       QT_TRANSLATE_NOOP("action","Hide All Recognizing Glyphs"),
+       QT_TRANSLATE_NOOP("action","Toggle 'Hide All Recognizing Glyphs'"),
```

Comment:
Maybe `Recognition Glyphs`? (not sure about the exact wording though)

Reply:
fixed

Reply:
Or "Recognized Glyph"?

Reply:
removed this code